[ADD] Cassandra backend for workload dashboard

bhaskarhc opened the pull request:
### What this pr ?
- Cassandra Connection using "cassandra-driver" npm package .
- Database Query to read and write user data in Cassandra .
- [add] -Deployment Yaml link added in yaml.js .

### Connection successful : 
```

|Cassandra-Jiva Connected . !
|__> keyspace `maya` created .
|___> Table `person` created .

```
### Read & Write Query:
![screenshot from 2018-11-12 15-18-58](https://user-images.githubusercontent.com/40464957/48341507-3a730e80-e693-11e8-93a7-5a30966863a8.png)

 

Signed-off-by: bhaskarjb <hcbhaskar7@gmail.com>

inyee786 commented on database/cassandra-jiva/dbconnection.js:
```diff
@@ -0,0 +1,31 @@
+const Uuid = require('cassandra-driver').types.Uuid;
+const cassandra = require('cassandra-driver');
+const PlainTextAuthProvider = cassandra.auth.PlainTextAuthProvider;
+const client = new cassandra.Client({ contactPoints:['localhost:9042'],authProvider: new PlainTextAuthProvider('cassandra', 'cassandra')});
```
localhost:9042 ??

inyee786 commented on database/cassandra-jiva/dbconnection.js:
```diff
@@ -0,0 +1,31 @@
+        var query = "CREATE KEYSPACE IF NOT EXISTS maya WITH replication = {'class':'SimpleStrategy', 'replication_factor' : 3};"
+        client.execute(query,function(err) {
+        if (!err) {
+            console.log("|__> keyspace `maya` created .");
```
give more generic log everywhere

inyee786 commented:
deploy the application first

bhaskarhc committed:
[ADD] Cassandra backend for workload dashboard

    -Cassandra Connection using "cassandra-driver" npm package .
    -Database Query to read and write user data in Cassandra .
    -[add] -Deployment Yaml link added in yaml.js .

Signed-off-by: bhaskarjb <hcbhaskar7@gmail.com>

chandankumar4 commented:
Need to send this PR again!!
